index.js

```diff
@@ -18,15 +19,15 @@ module.exports = (cypressConfig, dotEnvConfig) => {
   enhancedConfig.env = enhancedConfig.env || {}
 
   // get the name of all env vars that relate to cypress
-  const cypressEnvVarKeys = Object.keys(envVars).filter(envName => envName.startsWith('CYPRESS_'))
+  const cypressEnvVarKeys = all
+    ? Object.keys(envVars).filter(envName => envName)
```

Comment:
since nothing needs to be filtered out here, this could be simplified to just be `envName`

Reply:
@morficus oh, sorry, missed that. Updated the PR.